redwood-lazylayout-uiview/src/commonMain/kotlin/app/cash/redwood/lazylayout/uiview/UIViewLazyList.kt

```diff
@@ -275,6 +276,7 @@ internal class UIViewLazyList :
       rowHeight = UITableViewAutomaticDimension
       separatorStyle = UITableViewCellSeparatorStyleNone
       backgroundColor = UIColor.clearColor
+      contentInsetAdjustmentBehavior = UIScrollViewContentInsetAdjustmentNever
```

Comment:
Without this, `UIScrollView` adds the appropriate amount of space after the last element automatically, and we get double the space that we want.